### PR TITLE
Update to Chapter DA _ 30_differential_abundance.qmd

### PR DESCRIPTION
**Pull request by jkc9886**
1. Removed the two methods : LinDA and ZicoSeq
2. Working on the text and descriptions of the chapter.

### CODE REVIEW

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Did you build OMA locally and view that the chapter looks ok?

File: inst/pages/30_differential_abundance.qmd [hunk 1]
```diff
@@ -331,89 +331,6 @@ This will create a folder that is called like in the output specified
...
-microbiome compositional data (@Zhou2022). This is very similar to
-ANCOMBC with few differences: 1) LinDA corrects for the compositional
-bias differently using the mode of all regression coefficients. 2) it
-is faster (100x-1000x than ANCOMBC and according to the authors); 3)
-it supports hierarchical models. The latest ANCOMBC versions are also
-supporting hierarchical models. Nevertheless, LinDA seems a promising
-tool that achieves a very good power/fdr trade-off together with
-ANCOMBC according to the review. The speed improvements might make it
-critical especially for datasets that have higher sample or feature
-set sizes.
```
Comment (antagomir) [inline inst/pages/30_differential_abundance.qmd:346]: You could keep this explanation and move it to the end of the section under subtitle: "Further notes on differential abundance". Modify a bit to make the text fit the fact that the examples were removed.

File: inst/pages/30_differential_abundance.qmd [hunk 2]
```diff
@@ -331,89 +331,6 @@ This will create a folder that is called like in the output specified
...
-and permutation. Further details can be found in
-[this tutorial](https://cran.r-project.org/web/packages/GUniFrac/vignettes/ZicoSeq.html).
-This approach has been assessed to exhibit high power and a low false discovery
-rate, which has the following components:
-
-  1. Winsorization to decrease the influence of outliers;
-  2. Posterior sampling based on a beta mixture prior to address
-  sampling variability and zero inflation;
-  3. Reference-based multiple-stage normalization to address
-  compositional effects;
```
Comment (antagomir) [inline inst/pages/30_differential_abundance.qmd:382]: This explanation you can also move in a new subsection in the end.

File: inst/pages/30_differential_abundance.qmd (hunk 1 above)
Comment (jkc9886) [reply inst/pages/30_differential_abundance.qmd:346]: Sure, noted.

**Comment by jkc9886** [comment]
> Did you build OMA locally and view that the chapter looks ok?

Yes so far, I will build and test again once I edit the other text.

**Comment by TuomasBorman** [comment]
This is not linked to any issue, and the description does not say that. --> it would be nice to have short explanation why something is done, so we remember this also in the future

**Comment by TuomasBorman** [comment]
Closing because duplicated with https://github.com/microbiome/OMA/pull/508

**Pull request closed without merging**